src/Main.java

```diff
@@ -1,5 +1,26 @@
+import java.io.*;
+import java.util.ArrayList;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        try (FileOutputStream file = new FileOutputStream("test.txt");) {
+            String test = "i've checked your tests";
+            file.write(test.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        try (FileOutputStream file2 = new FileOutputStream("test2.txt");
+             FileInputStream file1 = new FileInputStream("test.txt");) {
+            int byteNumber;
+            ArrayList<Integer> array = new ArrayList<>();
+            while ((byteNumber = file1.read()) != -1) {
+                array.add(byteNumber);
+            }
+            for (int i = array.size()-1; i >= 0; i--) {
```

Comment:
This approach will not work for big files. See https://docs.oracle.com/javase/9/docs/api/java/io/RandomAccessFile.html